scripts/docker-compose/imports/openklant-database/database/1-setup-applicatie.sql

```diff
@@ -23,15 +25,25 @@ INSERT INTO public.klantinteracties_partij (id, bezoekadres_nummeraanduiding_id,
     VALUES (2, '', '', '', '', '', '', '', '', '', '', '3e9daeaa-e836-4450-aba7-2072477991ba', '0000000002', 'note', 'organisatie', false, 'dut', true, null, null);
 INSERT INTO public.klantinteracties_partijidentificator (id, uuid, andere_partij_identificator, partij_identificator_code_objecttype, partij_identificator_code_soort_object_id, partij_identificator_object_id, partij_identificator_code_register, partij_id)
     VALUES (2, 'ea241812-bcb1-4854-b814-6a32c0f1d2f5', '', 'KVK-NUMMER ONDERNEMING', 'Vestigingsnummer', '000012345678', 'KvK', 1);
+INSERT INTO public.klantinteracties_partij (id, bezoekadres_nummeraanduiding_id, bezoekadres_adresregel1, bezoekadres_adresregel2, bezoekadres_adresregel3, bezoekadres_land, correspondentieadres_nummeraanduiding_id, correspondentieadres_adresregel1, correspondentieadres_adresregel2, correspondentieadres_adresregel3, correspondentieadres_land, uuid, nummer, interne_notitie, soort_partij, indicatie_geheimhouding, voorkeurstaal, indicatie_actief, voorkeurs_digitaal_adres_id, voorkeurs_rekeningnummer_id)
+    VALUES (3, '', '', '', '', '', '', '', '', '', '', 'e6c36352-49c4-4fa9-9c24-98a9a2c30a2b', '0000000003', 'note', 'organisatie', false, 'dut', true, null, null);
+INSERT INTO public.klantinteracties_partijidentificator (id, uuid, andere_partij_identificator, partij_identificator_code_objecttype, partij_identificator_code_soort_object_id, partij_identificator_object_id, partij_identificator_code_register, partij_id)
+    VALUES (3, '605cbcb8-7508-4dca-9574-6dd5d5685fca', '', 'KVK-NUMMER ONDERNEMING', 'Vestigingsnummer', '000037143557', 'KvK', 1);
```

Comment:
Locally (Docker Compose) I think it is better to use explicitly fake KVK vestigingsnummers instead of 'real' ones. Why do you want to add this KVK number to our Docker Compose data? The JIRA task is about adding data to our test environment, not to our Docker Compose data. We should not confuse the two.

Reply:
Mainly because I wanted to have somewhere the insert statements documented. Provisioning repo is a better fit if we want to do that indeed.